Documentation/ExtensionArchitecture/Extbase/Reference/Domain/_Content.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyVendor\MyExtension\Domain\Model;
+
+use TYPO3\CMS\Extbase\DomainObject\AbstractEntity;
+
+class Content extends AbstractEntity
```

Comment:
why is this class not final?

Reply:
Hmm, thought about it. But not sure as we should propose models as final. But as you have the same opinion, okay,

Reply:
Mmh that is true, models are probably something you always want to be able to extend